e2e_testing/main.py

```diff
@@ -72,6 +72,10 @@ def _get_argparse():
     parser.add_argument("--crashing_tests_to_not_attempt_to_run_and_a_bug_is_filed",
                         metavar="TEST", type=str, nargs="+",
                         help="A set of tests to not attempt to run, since they crash and cannot be XFAILed.")
+    parser.add_argument("--ignore_failures", 
+                        default=False,
+                        action="store_true",
+                        help="return exit code 0 even if the test fails to unblock pipeline")
```

Comment:
I understand the utility of this flag to figure out which tests fail, but I wonder if we should drop it once this PR is ready to be merged.

Reply:
Currently it is only used because we have some unexpected passes with the stable torch version. I think this should be replaced to something with finer granularity. If you are ok with it, I would like to do this in a follow up PR

Reply:
Sounds good.